packages/ember-htmlbars/tests/node-managers/view-node-manager-test.js

```diff
@@ -0,0 +1,63 @@
+import ViewNodeManager from 'ember-htmlbars/node-managers/view-node-manager';
```

Comment:
:P *Pssst. There's component-node-manager too.*

I was thinking this should be pulled out to another PR so it doesn't interfere with the [BUGFIX release].